dg/media/configs.js

```diff
@@ -1149,6 +1258,8 @@ function() {
     var misc = {
         download_chunk_size: 2000,
         data_transfer_schema: "uploadqueue",
+        agg_variable: "2",
```

Comment:
we cant have agg and health specific values, as it is a MIS framework we need to keep config variables generic. We can have a dict instead